docs/migrate/backend_run_migration_guide.rst

```diff
@@ -0,0 +1,98 @@
+Migration guide: Migrate ``backend.run()`` from ``qiskit_ibm_provider`` to ``qiskit_ibm_runtime``
+=================================================================================================
+
+The Qiskit Runtime interface includes two packages:
+Qiskit IBM Provider (the ``qiskit_ibm_provider`` package) and
+Qiskit IBM Runtime (the ``qiskit_ibm_runtime`` package). Until now,
+Primitives (``Sampler`` and ``Estimator``)
+were run in Runtime. Custom circuits that performed their own transpilation and used ``IBMBackend.run()``
+were run in Provider.
+
+In this release, we add support for running custom circuits using ``IBMBackend.run()`` in Runtime,
+so users can run all programs through Runtime.
+We eventually plan to deprecate Provider.
```

Comment:
I would leave this note out, as I don't think we've officially announced this yet.

Reply:
@drew-distefano - Removed the last line. Please have a look.